zndraw/zndraw.py

```diff
@@ -102,46 +111,43 @@ def reconnect(self) -> None:
         self.socket.sleep(0.1)
         log.critical("Reconnected to server")
 
+    @typecast_kwargs
+    def get_data(self, **data: RoomGetData) -> RoomGetData:
+        with self._lock:
+            self._data = None
+            self.socket.emit("room:get", data.to_dict())
+            while self._data is None:
+                self.socket.sleep()
+                # generous timeout
```

Comment:
I assume `self.socket.sleep()` defaults to 1 second? I like the explicit arg though